topo/topo.go

```diff
@@ -453,6 +453,26 @@ func (m *Manager) Node(nodeName string) (node.Node, error) {
 	return n, nil
 }
 
+type resourcer interface {
+	Load(context.Context) error
+	Resources(context.Context) (*Resources, error)
+}
+
+// fakeTopology is used for testing only.
+type fakeTopology struct {
```

Comment:
you can define this in the _test.go file

edit: didn't see its use in topo params